server/test/handlers/inputSchool.test.js

```diff
@@ -0,0 +1,110 @@
+const schoolHandler = require('../../models/handlers/school');
```

Comment:
Hi Simran.
I would recommend renaming this file to `getWorkersForSchool` or something along that line to avoid confusion as these tests are not inputting school values.